Postfix/postfix/ingest/parser.yml

```diff
@@ -355,6 +355,10 @@ stages:
           network.protocol: "smtp"
         filter: "{{event.message.POSTFIX_SMTPD_CONNECT != null and event.message.tmp_ESMTP != null}}"
 
+      - set:
+          network.protocol: "smtp"
+        filter: "{{original.get('log', {}).get('syslog', {}).get('appname', '') in ('postfix', )}}"
```

Comment:
what does this part mean `in ('postfix', )` ? I did not understand the syntax

Reply:
changes made to make it work for events without `appname` were affecting `appname=postfix` events, so I had to make sure we have this field set properly for them. 

i used this specific construction with `get()` just to avoid a really long filter string. we don't need `in` though, I'll replace with `==`

Reply:
re: https://github.com/SEKOIA-IO/intake-formats/commit/bc0d2a25deba94702955996eb3fd7ee5e766309a - `POSTFIX_DISCARD` doesn't cover enough patterns for the case, while `POSTFIX_LOCAL` do